setup.py

```diff
@@ -13,6 +13,11 @@
 
 from setuptools import setup
 
+# Make it explicit that 32-bit systems are not supported
+if platform.architecture()[0] == '32bit':
+    print("[!] Pwntools does not support 32-bit Python. Use a 64-bit release.")
```

Comment:
Please add the comment from the warning in [`pwn/__init__.py`](https://github.com/Gallopsled/pwntools/blob/dd1edaa6c9f2b2db3a5ea91bac42cb9940fc0b6f/pwn/__init__.py#L12-L16) as well.